src/robust_provider/provider.rs

```diff
@@ -384,34 +351,41 @@ impl<N: Network> RobustProvider<N> {
         Fut: Future<Output = Result<T, RpcError<TransportErrorKind>>>,
     {
         let num_fallbacks = self.fallback_providers.len();
-        if num_fallbacks > 0 && start_index == 0 {
-            info!("Primary provider failed, trying fallback provider(s)");
-        }
 
         let fallback_providers = self.fallback_providers.iter().enumerate().skip(start_index);
         for (fallback_idx, provider) in fallback_providers {
             if require_pubsub && !Self::supports_pubsub(provider) {
-                info!(
-                    fallback_index = fallback_idx + 1,
+                trace!(
+                    provider_num = fallback_idx + 1,
                     "Fallback provider doesn't support pubsub, skipping"
                 );
                 continue;
             }
-            info!(fallback_index = fallback_idx + 1, "Attempting fallback provider");
+
+            tracing::trace!(
+                fallback_provider_index = fallback_idx + 1,
+                total_num_fallbacks = num_fallbacks,
+                "Attempting fallback provider"
+            );
```

Comment:
```suggestion
           trace!(
                fallback_provider_index = fallback_idx + 1,
                total_num_fallbacks = num_fallbacks,
                "Attempting fallback provider"
            );
```
this is a regular feature-gated trace